### DISCUSSION TITLE
Can't find a way to make ImageGlass my Default Image Viewer in Windows 10

### DISCUSSION THREAD

**Discussion by SufyanAbbada** [d2phap/ImageGlass#1037] [Q&A] [answered]
### System information:
 - Windows OS version: 10
 - ImageGlass version: Latest (8.1.4.18)

### Actual behavior:
There is no way to make ImageGlass as my Default Image Viewer once I download it. Even in Windows, it is not showing in the available applications

### Expected behavior:
There should be an option for making it a default Image Viewer application even after installing it.

**Comment by silverwings15** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10742312/116795762-83d4a400-ab01-11eb-8b2d-ffcb750158cd.png)

![image](https://user-images.githubusercontent.com/10742312/116795770-8c2cdf00-ab01-11eb-9515-8d7271367348.png)
